spec/models/podcast_spec.rb

```diff
@@ -54,4 +54,26 @@
       it { is_expected.to be_nil }
     end
   end
+
+  describe 'validations' do
+    it 'validates the presence of a slug' do
+      valid_podcast = build(:podcast, slug: 'anarchy')
+
+      expect(valid_podcast).to be_valid
+    end
+
+    it 'validates lack of presence of a slug' do
+      invalid_podcast = build(:podcast, slug: nil)
+
+      expect(invalid_podcast).not_to be_valid
+    end
+
+    it 'validates uniqueness of slug' do
+      valid_podcast = build(:podcast, slug: 'anarchy')
+      invalid_podcast = build(:podcast, slug: 'anarchy')
```

Comment:
```suggestion
      invalid_podcast = build(:podcast, slug: 'anarchy')
      
```

Just a little blank space for readability.